dev-docs/bidders/adview.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bidder
+title: AdView
+description: Prebid AdView Bidder Adapter
+biddercode: AdView
+gdpr_supported: true
+gvl_id: 1022
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: false
+userId: 
+media_types: banner, video, native
+safeframes_ok: true
+bidder_supports_deals: true
+pbjs: false
+pbs: true
+pbs_app_supported: true
+prebid_member: false
+---
+
+### Note:
+
+The Example Bidding adapter requires setup before beginning. Please contact us at partner@adview.com
```

Comment:
Please include a note that this adapter does not support multiple impressions and does not perform impression splitting, such that publishers clearly understand only the first impression is processed and the rest are ignored.

Reply:
ok, i have committed new doc include the notes. pls check it.

Reply:
Sorry for the late response, by the way, if all goes well, could you help share the update date of the "Bid Adapter List" on the Prebid website?